x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java

```diff
@@ -353,7 +353,7 @@ private static SearchResponse mergeFinalResponse(
             totalShards,
             sucessfulShards,
             skippedShards,
-            took,
+            took, // TODO: since this is combining multiple pre-existing search responses, we don't want to measure twice.
```

Comment:
@javanna I'm hoping to confirm my understanding that we don't need instrumentation here, as "took"times should have already been recorded by the individual responses prior to this point, correct?

Reply:
Good question. I think you are correct.